docs/experiments-covid.md

```diff
@@ -64,6 +64,19 @@ And the effectiveness of the above runs:
 | `anserini` | `r3.fusion2` | 0.6100 | 0.9100 | 0.2641
 | `anserini` | `r3.rf`      | 0.6883 | 0.9750 | 0.2817
 
+**(Updates 2020/06/23)** We report the effectiveness of all the runs that were prepared _for_ round 3 using the cumulative qrels file (containing qrels from round 1, 2 and 3) mapped to the Round 3 (May 19) release of CORD-19. 
```

Comment:
To be absolutely explicit, say something like:

This is the qrels file named `blah` by NIST (add link to URL), which is stored in our repo as [filename](link)....